### PR TITLE
Add documentation

### PR DESCRIPTION
**Pull request by jonlamdev**
* Addresses #47, for adding standardized repo documentation and updating the README
* Addresses #49 to initialize the develop documentation. The WIP is deployed: https://statcan.github.io/StatCan.OrchardCore/ 
* todo: Module documentation will need to be updated in a separate ticket ; localization

### CODE REVIEW

**Comment by andrewgarneau** [comment]
A few things I would suggest (coming from a student with no dotnet experience). These are definitely coming from someone with less familiarity with our projects, but if the team wants to keep recruiting new people and students, it's important to make the documentation clear and self explanitory

- NuGet: Mention where to find the `artifactory credentials` that you need to add to the config file
- NuGet: Is the file location of the NuGet config on the system, or part of the project once it's cloned? Maybe linking to the repo or referencing it by name would be useful
- Recommended PowerShell setup: It might be a good idea to include what the bash or zsh commands are in addition to the PowerShell ones, unless assumed everyone is developing on Windows. Again I'm a bit lost in the sense of the context of this as mentioned in the last point.

**Comment by arnaudroystatcan** [comment]
Missing the hackathon module/theme? Other than that I think it looks good so far.

**Comment by mfan127** [comment]
Maybe adding a section for release notes or change logs.

File: docs/en/development/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+  </packageSources>
+</configuration>
+```
+
+
+### Recommended VSCode extensions
+
+- Omnisharp (C#)
+- Liquid Language Support
+- Bracked pair colorizer
```
Comment (MackenzieBarrett) [inline docs/en/development/getting-started.md:64]: Typo. "Bracket pair colorizer"

File: docs/en/development/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+function ibp { 
+  ib
+  dotnet .\bin\Debug\netcoreapp3.0\Inno.Web.dll
+}
+```
+Don't forget to restart your powershell session to load the changes.
+
+## Quickstart 
+
+- Run this ese commands in powershell:
```
Comment (MackenzieBarrett) [inline docs/en/development/getting-started.md:128]: Typo? I expect his is supposed to be 'these'

File: docs/en/development/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+
+You can also run `yarn run watch` to rebuild assets when changes are detected.
+
+Supported types: SaSS, ts, js, and Less.
+
+## Testing
+
+The framework used to test the innovation website is Cypress.
+
+Note: Running the tests assume a clean environment and may fail if you run them on an existing instance.
```
Comment (MackenzieBarrett) [inline docs/en/development/getting-started.md:159]: A little awkward. Recommend changing to "Note: The tests assume..."

File: docs/en/development/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+
+To run tests:
+
+- From the `test` folder of the project, run `yarn install` (if you have yarn installed) or `npm install`
+- From the same folder, you can run `yarn cypress` to open cypress and start running tests
+
+## Debugging
+
+We include a vscode `launch.json` / `tasks.json` files to help debug your code.
```
Comment (MackenzieBarrett) [inline docs/en/development/getting-started.md:168]: "We include a {...} files". Recommend changing to 

> We include `launch.json` / `tasks.json` files to help debug your code.

File: docs/en/development/localization.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+Orchard uses [PO](https://www.gnu.org/software/gettext/manual/html_node/PO-Files.html) files to support translation / pluralization of static strings.
+
+See the [orchard documentation](https://orchardcore.readthedocs.io/en/dev/docs/guides/install-localization-files/).
+
+## How to generate POT files from our modules
+
+1. Install the [po-extractor](https://github.com/lukaskabrt/PoExtractor) tool. `dotnet tool install --global PoExtractor.OrchardCore`
+2. Run `extractpo-oc C:\PathToRepo\src C:\temp\OrchardInno --liquid`
+
+## Where to place translated PO files
```
Comment (MackenzieBarrett) [inline docs/en/development/localization.md:12]: Missing section?

File: docs/en/reference/modules/GitHub.md [hunk 6]
```diff
@@ -0,0 +1,3 @@
+# GitHub (`StatCan.OrchardCore.GitHub`)
+
+This module adds feature to interact with GitHub to OrchardCore. This is different than the authentication module [`OrchardCore.GitHub`](https://docs.orchardcore.net/en/dev/docs/reference/modules/GitHub/)
```
Comment (MackenzieBarrett) [inline docs/en/reference/modules/GitHub.md:3]: Somewhat awkward phrasing. Would recommend 

> This module adds the ability to interact with GitHub using OrchardCore.

File: docs/en/reference/themes/PortalTheme.md [hunk 7]
```diff
@@ -0,0 +1,3 @@
+# Digital Admin Theme
```
Comment (MackenzieBarrett) [inline docs/en/reference/themes/PortalTheme.md:1]: Copy/paste of DA theme doc

File: docs/mkdocs.yml [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+             - Matomo: reference/modules/Matomo.md
+             - Scripting: reference/modules/Scripting.md
+             - Workflows: reference/modules/Workflows.md
+        - Themes:
+             - DigitalAdmin: reference/themes/DigitalAdmin.md
+             - PortalTheme: reference/themes/PortalTheme.md
+    - Support:
+        - Contact: contact.md
+
+copyright: 'Copyright &copy Her Majesty the Queen in Right of Canada, as represented by the Minister of Statistics Canada, 2020'
```
Comment (MackenzieBarrett) [inline docs/mkdocs.yml:39]: Is this &copy intentional?

Comment (jonlamdev) [reply docs/mkdocs.yml:39]: The copyright mark

**Comment by jptissot** [comment]
@jonlamdev how are we dowin on this one ?

**Comment by jonlamdev** [comment]
Closing...merging to master as it is "minimally viable": there is hosted documentation, tracked in source, and mechanism for updating. Content needs to be updated and kept updated.

**Comment by jonlamdev** [comment]
Thanks all for comments and first look 🎉

**Pull request merged**